CMakeLists.txt

```
@@ -161,6 +164,7 @@ set (OPENCL_COMPILE_DEFINITIONS
     OPENCL_ICD_LOADER_VERSION_MINOR=0
     OPENCL_ICD_LOADER_VERSION_REV=6
```

Sorry, one more suggestion.  Since this is a pretty significant change, could you rev the ICD loader version?

```suggestion
    OPENCL_ICD_LOADER_VERSION_REV=7
```

---

Thanks! Good catch.